clj-refactor.el

```diff
@@ -724,7 +724,7 @@ at the opening parentheses of an anonymous function."
           (when (<= (point) search-bound)
             (error "Can't find definition of anonymous function!")))))))
 
-(defun cljr--even? (n)
+(defun cljr--even-p (n)
```

Comment:
This should've been `cljr--evenp`.  There's only a hyphen before the `p` if there's a hyphen in the predicate.

See more [here](https://www.cs.cmu.edu/Groups/AI/html/cltl/clm/node69.html) if you're **really** interested.